kodilla-testing2/src/test/java/com/kodilla/testing2/crudapp/CrudAppTestSuite.java

```diff
@@ -0,0 +1,145 @@
+                    buttonCreateCard.click();
+                });
+
+        Thread.sleep(5000);
+        Alert alert = driver.switchTo().alert();
+        if (alert.getText().contains("Card created ")) {
+            alert.accept();
+        }
+    }
+
+    private boolean checkTaskExistsInTrello(String taskName) throws InterruptedException {
+
+        final String TRELLO_URL = "https://trello.com/login";
+        boolean result = false;
+
+        WebDriver driverTrello = WebDriverConfig.getDriver(WebDriverConfig.CHROME);
+        driverTrello.get(TRELLO_URL);
+
+        driverTrello.findElement(By.id("user")).sendKeys("wjola");
+        driverTrello.findElement(By.id("password")).sendKeys("Z?a4,pqk&vb]t[X");
+        WebElement el = driverTrello.findElement(By.id("login"));
+        el.submit();
+
+        Thread.sleep(4000);
+
+        driverTrello.findElement(By.id("username")).sendKeys("jola-w@o2.pl");
+        driverTrello.findElement(By.id("login-submit")).submit();
+
+        Thread.sleep(1000);
+
+        driverTrello.findElement(By.id("password")).sendKeys("Z?a4,pqk&vb]t[X");
+        driverTrello.findElement(By.id("login-submit")).submit();
+
+        Thread.sleep(3000);
+
+        driverTrello.findElements(By.xpath("//a[@class=\"board-tile\"]")).stream()  // [9]
+                .filter(aHref -> aHref.findElements(By.xpath(".//div[@title=\"Kodilla Application\"]")).size() > 0)
+                .forEach(WebElement::click);
+
+        Thread.sleep(4000);
+
+        result = driverTrello.findElements(By.xpath("//span")).stream()
+                .anyMatch(theSpan -> theSpan.getText().equals(taskName));
+
+        driverTrello.close();
+
+        return result;
+    }
+
+    private boolean canTaskDeleteFromDatabase(String taskName) throws InterruptedException {
+
+        boolean result;
+
+        while (!driver.findElement(By.xpath("//select[1]")).isDisplayed());
+        driver.findElements(By.xpath("//form[@class='datatable__row']")).stream()
+                .filter(row ->
+                        row.findElement(By.xpath(".//p[@class='datatable__field-value']"))
+                                .getText().equals(taskName))
+                .forEach(taskNameRow -> taskNameRow.findElement(By.xpath(".//button[text()='Delete']")).click());
+        Thread.sleep(4000);
+
+        result = driver.findElements(By.xpath("//form[@class=\"datatable__row\"]")).stream()
+                .anyMatch(anyForm -> anyForm.findElement(By.xpath(".//p[@class=\"datatable__field-value\"]"))
+                        .getText().equals(taskName));
+        Thread.sleep(4000);
+        return  result;
+    }
+
+    @Test
+    void shouldCreateTrelloCard() throws InterruptedException {
+        String taskName = createCrudAppTestTask();
+        sendTestTaskToTrello(taskName);
+        assertTrue(checkTaskExistsInTrello(taskName));
+        assertFalse(canTaskDeleteFromDatabase(taskName));
```

Comment:
dlaczego to jest asercja? Dlaczego metoda sie nazywa canTaskDelete.... To ma byc metoda sprzatajaca po tescie